x-pack/plugins/security_solution/public/management/pages/event_filters/view/components/form/index.tsx

```diff
@@ -90,11 +90,12 @@ export const EventFiltersForm: React.FC<EventFiltersFormProps> = memo(
     const handleOnChangeName = useCallback(
       (e: React.ChangeEvent<HTMLInputElement>) => {
         if (!exception) return;
+        const name = e.target.value.toString().trim();
```

Comment:
I noticed from the GIF that when the flyout opens, the name field is already marked as "error". Not sure if that was like that before - if so, then please disregard. but (at least with TA) we tried to ensure we don't light up the form with errors until the user has had a chance to "visit" the field. So we only show errors for a given field after they have left that field (`.blur()` event).

Reply:
Yeah! This is fixed here: https://github.com/elastic/kibana/pull/101563 -> https://github.com/elastic/kibana/pull/101563/files#diff-c8d2872401b39102ce1b83c8750bee6a23e3a78d7fc5a6f52fe02b08520ae603R159